walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/VerifiedIdPartialRequest.kt

```diff
@@ -0,0 +1,79 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+package com.microsoft.walletlibrary.requests
+
+import com.microsoft.walletlibrary.requests.requirements.GroupRequirement
+import com.microsoft.walletlibrary.requests.requirements.Requirement
+import com.microsoft.walletlibrary.requests.requirements.VerifiedIdRequirement
+import com.microsoft.walletlibrary.requests.styles.RequesterStyle
+import com.microsoft.walletlibrary.requests.styles.VerifiedIdStyle
+
+/**
+ * Represents an incomplete mutable VerifiedID Request for RequestProcessorExtensions to modify.
+ */
+data class VerifiedIdPartialRequest(
+    /**
+     * Display information for the requester
+     */
+    var requesterStyle: RequesterStyle,
+
+    /**
+     * Potential display information for the Verified ID being issued (if this is an issuance request)
+     */
+    var verifiedIdStyle: VerifiedIdStyle?,
```

Comment:
VerifiedIdPartialRequest represents a mutable form of the request that was parsed, so it extends all fields. The style is not expected to be populated on presentations. The intent is if an extension for additional style properties is injected into the client, it could parse and enrich the style.